assets/js/frontend/edd-ajax.js

```diff
@@ -21,7 +21,8 @@ jQuery( document ).ready( function( $ ) {
 				cart_item: item,
 				nonce: nonce,
 				timestamp: $this.data( 'timestamp' ),
-				token: $this.data( 'token' )
+				token: $this.data( 'token' ),
+				current_page: edd_scripts.current_page,
```

Comment:
I went off of the same variable that's being used for the `url` property so that the function here is internally consistent and so I know I'm using a localized variable that is for sure defined--EDD has a couple different ones that it loads and although there's a lot of replication, they are condi
tionally used.